components/registry-facade/pkg/registry/manifest.go

```diff
@@ -495,6 +505,16 @@ func DownloadManifest(ctx context.Context, fetch FetcherFunc, desc ociv1.Descrip
 
 	switch rdesc.MediaType {
 	case images.MediaTypeDockerSchema2Manifest, ociv1.MediaTypeImageManifest:
+		// ! in case of mismatch between platform and manifest OS - arch we'll throw an error. Platform can be nil so handle accordingly
+		if rdesc.Platform != nil {
+			log.Info("DS: 5: ", rdesc.Platform.OS, "-", rdesc.Platform.Architecture, "")
+			if fmt.Sprintf("%s-%s", rdesc.Platform.OS, rdesc.Platform.Architecture) != fmt.Sprintf("%s-%s", runtime.GOOS, runtime.GOARCH) {
+				log.Info("DS: 6: ", rdesc.Platform.OS, "-", rdesc.Platform.Architecture)
```

Comment:
6?

Reply:
remove all the numbers from log statements, don't make sense in a production codebase

Reply:
Alright

Reply:
Removed this part altogether since we are not handling the image mismatch edge case in registry facade anymore.